samples/Mvc.Server/Startup.cs

```diff
@@ -46,6 +47,18 @@ public void ConfigureServices(IServiceCollection services)
                 options.ClaimsIdentity.RoleClaimType = Claims.Role;
             });
 
+            // OpenIddict offers native integration with Quartz.NET to perform scheduled tasks
+            // (like pruning orphaned authorizations/tokens from the database) at regular intervals.
+            services.AddQuartz(options =>
```

Comment:
This `AddQuartz` registers also the Microsoft logging "bridge", a.k.a. `MicrosoftLoggingProvider`  which will enable Quartz to log to global logger factory. This will affect all schedulers build, so it's global and not scheduler factory global. If you would take the route to have scheduler per tenant then we would need `options.UseLoggerFactory(yourScopedFactory);`?

Reply:
If you wanted to support having multiple scheduler factories in the DI container but wanted a different logger factory for each, yeah, that would be a nice option. Now, if you implement multi-tenancy a-la-Orchard-Core, it manages an `IServiceProvider` per tenant, so each tenant would get a unique scheduler and a unique logger factory.